MekHQ/data/scenariomodifiers/FacilityAlliedDefend.xml

```diff
@@ -1,5 +1,5 @@
 <AtBScenarioModifier>
-    <additionalBriefingText>Preserve 50% of the designated turrets.</additionalBriefingText>
+    <additionalBriefingText>Preserve 25% of the designated turrets.</additionalBriefingText>
```

Comment:
This change was to hold base defenses to a similar standard as base assaults. Previously, players were required to keep 50% of the base turrets alive, while they were required to kill 75-100% to destroy an enemy base.